src/commands/inputs.rs

```diff
@@ -0,0 +1,96 @@
+use anyhow::Result;
+use clap::Parser;
+use serde_json::{Value, json};
+use std::path::PathBuf;
+use wdl::{
+    ast::{
+        AstToken, Document, SyntaxKind, Visitor,
+        v1::{
+            self,
+            Expr::{self, Literal},
+            LiteralExpr, LiteralNone, Type,
+        },
+    },
+    doc,
+    grammar::SyntaxTree,
+};
+
+#[derive(Parser, Debug)]
+#[command(about = "Generate input JSON from a WDL document", version, about)]
+pub struct InputsArgs {
```

Comment:
We want to support being able to construct an input JSON for both WDL workflows and tasks. This will require accepting a `name: Option<String>` argument. If the provided name doesn't match a task or workflow in the provided document, we should error. If no `name` is provided, we should assume the user wants to create inputs for the workflow in the document if it's present. If no workflow is in the document, we then assume there will be _one_ task defined in the entire document, and construct inputs for that task. If more than one task is present, we can error with a message instructing the user that a `name` must be provided for this WDL document.